resources/velero/values.schema.json

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Schema definition for velero helm chart values",
+    "type": "object",
+    "properties": {
+        "global": {

```

Comment:
This is outdated. Please update it before merging

Reply:
done, please check again